pkg/motor/resource.go

```diff
@@ -101,3 +104,43 @@ func (r *motorResources) GetSchema(did string) (*st.WhatIs, *st.SchemaDefinition
 
 	return nil, nil, false
 }
+
+func (r *motorResources) GetWhereIs(ctx context.Context, did string, address string) error {
```

Comment:
Consider checking in cache for the value before fetching

Reply:
`GetWhereIs` is responsible for updating the cache